.betterer.results

```diff
@@ -3874,10 +3874,26 @@ exports[`better eslint`] = {
     "public/app/features/dashboard/components/DashboardRow/DashboardRow.tsx:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
     ],
-    "public/app/features/dashboard/components/DashboardSettings/DashboardSettings.test.tsx:5381": [
+<<<<<<< HEAD
```

Comment:
think i'll have to look into how this passed CI 😬 

but definitely needs a `yarn betterer -u` committed + pushed

Reply:
fixed